flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkThroughputBenchmarkTests.java

```diff
@@ -52,6 +52,14 @@ public void largeRemoteMode() throws Exception {
 		env.tearDown();
 	}
 
+	@Test
+	public void largeRemoteAlwaysFlush() throws Exception {
+		StreamNetworkThroughputBenchmark env = new StreamNetworkThroughputBenchmark();
+		env.setUp(1, 1, 0, false);
+		env.executeBenchmark(1_000_000);
+		env.tearDown();
+	}
```

Comment:
Is this not tested via your non-IT tests now?
(we should be careful about adding integration/full stack tests because of their added time)

Reply:
This is smaller scope test compared to ITCase and it covers higher load. It's a very good stress test for investigating and detecting deadlocks/race conditions in the network stack.